ca-ES.lproj/Menu.strings

```diff
@@ -238,7 +238,11 @@
 "Copy Image to" = "Copia la imatge a";
 "Open Image Location" = "Obre la ubicació de la imatge";
 
+<<<<<<< HEAD
 "Learn More" = "Aprèn més";
+=======
+"Learn More" = "Més informació";
+>>>>>>> upstream/master
```

Comment:
Would you either use "Aprèn més" or use "Més informació" ?

Reply:
I just forgot to save this conflict.
This change was made in the previous PR. "Aprèn més" is the literal translation of "Learn more".
"Més informació" means "More information"; which is kind of more natural.